frontend/types/primaryDonor.ts

```diff
@@ -10,3 +10,13 @@ export type PrimaryDonorData = {
 export type PrimaryDonorPostData = WithoutId<PrimaryDonorData>;
 
 export type PrimaryDonorPutData = WithoutId<PrimaryDonorData>;
+
+export type CouponSponsorship = {
+  primaryDonor: PrimaryDonorData;
+  primaryDonorAmount: number;
+};
+
+export type CouponSponsorshipSummary = {
+  primaryDonorName: string;
+  couponDenomination: number;
+};
```

Comment:
should we use only one of these? seems like the info in CouponSponsorshipSummary can be provided through CouponSponsorship. Also maybe we can stick to one term: either primaryDonorAmount or couponDenomination since they refer to the same thing in our contexts?